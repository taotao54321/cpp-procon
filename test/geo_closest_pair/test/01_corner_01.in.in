2
-100.000 -100.000
100.000 100.000
