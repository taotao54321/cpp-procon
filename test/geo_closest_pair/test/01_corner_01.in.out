282.8427124746
